checkov/common/util/templates/resource.jinja2

```diff
@@ -0,0 +1,30 @@
+from lark import Token
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class {{title}}(BaseResourceCheck):
```

Comment:
@schosterbarak do we want to have `BaseResourceCheck` as the default one or change it to `BaseResourceValueCheck`, which is usually much easier to implement, but also not so flexible like the `BaseResourceCheck`?

Reply:
Perhaps this one should remain `BaseResourceCheck` and we could create a separate template and prompt for simple ValueChecks?

Reply:
@jmeredith16 your suggestion sounds good

Reply:
I'll have to get on that early next week. Sorry for the delay.